semcore/d3-chart/src/types/Area.d.ts

```diff
@@ -30,8 +30,8 @@ export type AreaProps = Context & {
 /** @deprecated */
 export interface IAreaDotsProps extends AreaDotsProps, UnknownProperties {}
 export type AreaDotsProps = Context & {
-  /** Show all Dot */
-  display?: boolean;
+  /** Show dots */
+  display?: boolean | ((index: number, active: boolean, single: boolean) => boolean);
```

Comment:
I think, we should pass a value here too

Reply:
I intentionally didn't passed it because we can't type it properly and index is totally enough to get value from initial data array